R/worker_apply.R

```diff
@@ -45,7 +45,10 @@ spark_worker_init_packages <- function(sc, context) {
   else {
     spark_env <- worker_invoke_static(sc, "org.apache.spark.SparkEnv", "get")
     spark_libpaths <- worker_invoke(worker_invoke(spark_env, "conf"), "get", "spark.r.libpaths", NULL)
-    if (!is.null(spark_libpaths)) .libPaths(spark_libpaths)
+    if (!is.null(spark_libpaths)) {
+      spark_libpaths <- unlist(strsplit(spark_libpaths, split = ","))
```

Comment:
I specifically used base R functions here since I think we can make no assumptions as of when this runs that any non-base packages will be available.

Reply:
That is correct... all the `worker_` and `core_` files can't depend on any packages since they are used in the worker nodes to load packages in `spark_apply()`.